src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,10 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import './CommentList.scss';
+import { CommentInfo } from '../CommentInfo';
+
+export const CommentList = ({ comments }) => (
+  <div className="CommentList">
+    {comments.map(comment => (
```

Comment:
No checks are performed to ensure that 'comments' is not null or undefined before mapping over it. This could potentially cause a runtime error if 'comments' is not an array.

Reply:
not solved